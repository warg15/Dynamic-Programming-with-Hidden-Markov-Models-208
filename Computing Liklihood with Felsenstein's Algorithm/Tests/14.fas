>Annas-hummingbird
CACTGCGGTTGCCGCAGATTCGGTTCGCTCATCTTTGTACTATAATTTGACTGACGCTACTAGCACTGTATAGTTAAGTACACATCAACTAAAAGAACGGTTCAACTCGAATTGGTAAAGCAATATCAAGGTCTTATCAGATAGTTCTCTGCTCCTGAGTAGATCTTAAGTTGTTGAGCCATTGATACTCGCAAAATGTT
>Crested-ibis
CACCACGGTGGCCGAAGATTCGGTTGGATCGTCCTTGCACTATGATTCGACAACCGCTACTAGCACTGTATAGTTAAATACACATCAACTAAAAGGACGGTTCCGCTCGAACAGGTACAGCAATATTAAGGTCTTATGAGATAGTTCTCTGCTTCTGAGTAGATCACAAGTTGCTGAGGCAATGATACCCGCGAAAGGTT
>Budgerigar
CACCACGTTGGCCGAAGATTCGGTTCGATCTTCTTTGTACTATAATTCGACAAGGGCTACTAGCACTGTATAGCTAAATACACATCTAATAAAAGGACGGTTCCGCTCGAACAGGTACAGCAATATTAGGGTCTTATGAGATGTTTCTCGGCTTTTGAGTAGAATACAAGTTGCTGAGACATTGATACCCGCCAAACGTT
>Adelie-penguin
CACCACGGTGGCCGCAGATTCGGTTGAATCGTCCTCGTACTATAATTCGACAACCGCTACTATCACTGTATAGTTAAATACACATCAACTGAAAGGACGGTTCAGCTCGAACAGGTACAGCAATATTAAGGTCTTATGAGATAGTTCTCTGCTTCTGAGTTGATCACAAGTTGCTGAGACAGTGATACCCGCAAAAGGTT
>Kea
CACCACGCTGGCCGATGATTCGGTACGATCTTCCTTTTGCTATAATTCGACAAGCGTCACTAGCACTGTATAGTTAAATACACATCCAATAAAAGGACGGTTCCTCTCGAATAGGTACAGCAATATTAAGGTCTTATGAGATAGTTCGCGGATTTTAAGTAGATCACAAGTTGCTGAGACATTGATACCCGCCAAACGTT
>Northern-fulmar
CATCACGGAGGCCGAAGATTCGGTTGGATCGTCCTTGTACTATAATTCGACAACCGCTACTAGCACTGTATAATTAAATACACATCAACTAAAAGGACGGTTCCGATCGAACAGGTACAGCAATATTAAGGTCTTATGAGATAGGTCTCTGCTTCTGAGTAGATCACAAGTTGCTGAGACAGTGGTACCCGCAAAAGGTT
>Hoatzin
CACCACGGCGGGCGAAGATTTGATTCGATCGTCCTTGTACTATAATTCGACAGGCGCTGCTAGCACTGTATAGTTAAATACACGTCAACTAAAAGGACGGTTCCGCTCGAACAGGTACAGCAATGTTAAGGTCTTATGAGATAGTTCTCTGCTCCTGAATAGATCACAAATTGCTGCGACATTGATACCCCCAAGAGGTT
>Chimney-swift
CACCACGGTGGCCGAAGATTCGGTTCGTTCGTCCTTGTAGTATAATTCAACAAACGCTACTAACACTGTGTATTTAAATACACATCAACTAAAAGTGGGGTTCCGCTCGAACAGGTACAGCAATATTAAGGTCTTATGAGATGGTTCTCTGCTCCTGAGTAGATCACATGTTGCTGAGACATTGATACCCGCGGACTGTT
>MacQueens-bustard
CACCACGGTGGCCGAAGATTAGGTTAGATCGTCCTTGTACTATAATTCAACAAGCGCTACCAGCACTGTATAGTTAAATACACATCAACTAAAAGGACGGTTCCGCTCGAACAAGTACAACAATATTAAGGTCTTATGAGATAGTTCCCTGCTCCTGATTGGATCACAAGTTGCTGAGACGTTGATACCCGCAAAAGGTT
>Cuckoo-roller
CATTACGGGGGCCGAAGATTCGGTTTGACCGTCCTTGTACTATAATTCGGCAAGCGCTACTAGCACTGTATAATGAAATACATATTAACTAAAAGAACGGTTCCGCTCGAACAGGTGCAGCAATATTAAAGTCTGACGAGCTAGTCCTCTGCTTCTGAGAAGATCACAAGTTGCTGAGACATTGATACCCGCAAAAGGTT
>Brown-mesite
CACCACGGTGGCCGAAGATTCGGTTTGATCGTCCTCGTACCAGAATTGGGCAAGCGCTATTAGTATTGTATGGTTAAATACACATCAACAAGAAGGACCGTTCCGTTCGAACAGGTACAGCAATATTAAGGGCTTATGGGATAGTTATCTGCCCCTGAGTAGATCACAAGTTGCTGAGACACTAATACCCACAAAAGGTA
>Pigeon
CATCACGGCGGCCGCACATTCGGTTCGATCGGCCTTGTACTATAATTCGACAAGCGCCACTAGCACTGTATAGTTAAATATACATCAACTAAAAGGACGGTTCCGCCCAAACAGGTACAGCAATATTAAGATCTTATGAGATAATTCTCTGGTCCTGAGTAGATCACAAGTTGCTGAGACATTGATACCCGCAAATGGTT
>Bald-eagle
CACCACGGTGGCCGAAGATTCGGTTCGATCGTCCTCGTACTATAATTCGACAAGCGCTACTAGCACTGTATAGTTAAATACACATCAACTAAAAGGACAGTTCCGCTCGAACAGCTACAGCAATATTAGGGTCTTATGAGATAGTTCTCTGCTACTGAGTAGATCACAAGTTGCTGAGACATTGATACCCGCAAAAGGTT
>Grey-crowned-crane
CACCACGATGGCCGAAGCTTCTGTTCGATCGTCCTCGTACTATGATTCGACAAGCGCTACTAGCACGGTATAGTTAAATACACATCAACTAAAAGGACGGTTCCGCTCGAACAGGTACAGCAATATTAAGGTCTTATGAGATAGTTCTCTGCTCCTGAGTAGATCAAAAGTTGCTGGGACATTGATACCCACAAAAGTCT
>Speckled-mousebird
CACCACGGAGGCCGAAAATTCGGTTCCATCGTACTTGTACCCTAATTCGACGAACGCCAGTAGTAGGGTGTAGTTAAACACGCATCACCTAATAGGACGGTTCCGCTCGAACAGCTACAGCAATATTAAAGTCTTATGAGATAGTTTTCTGCTTCTGAGTAGATCAAAAGTTGCGGAGACAATGATACCCGCAAAAAGTT
>White-tailed-tropicbird
CACCACGGTGGCCGAAGTTTCGGTCGGATCGTCCTCGTACTATAACTCGACAAGCGCTACTAGCACTGTATAGTTAAATACACATCAACTAAAAGGACGGTTACGTTCGAACAGGCACACCAATATTGACGTCTTATGAGATAGTTCCCTGCTTCCGAGTAGATCACAAGTTGCTGATACGTTGATAGCCGCAAAAGGTT
>Barn-owl
CACCACGGTGGCTGAAGATTAGGTTCGATCGTCCTTGTACTATAATTCGACAAGCGCCACTAGCACTGTATAGTTAAATACACATCAACTAAAAGGACGGTTCCGATCAAACAGGTACAGCAATATTAAGGTCTTATGAGATAGTTTTCTGCTTCTGAGTAGATAACAAGTTGCTGAGAGATTGCTACTCGCAAAGGGTT
>Red-crested-turaco
CGCCACGGTGGCCGAAGATTCGGTTCGATCGTCCTTGTACTATAAATCGACAAGCGCTACTAGCGCTGGATAGTTTAATACACATCAACTACAAGAACGGTTCCGCTCGAACAGGTACAGCTATATTAAGGTTTTATGAGATAGTTCTCCGCTCCTGAGTAGATCACAAGTTGCTGAGACATTGATACCCGCAAAAGGTT
>Pekin-duck
AACCACAGTGGGCAACCACCCGGTTTAATTGTCCTTATACTATAATTCAACGAGCGTTACTAACATTGTATAGTCAAATACACATCAACCGAAAGGACGGTTACGCACGGACGGGTACAGCAACATTAATGTTTTCTGAGATAGTTCTCTGCTCCTGGGTGGGTCACAAGCTGCTGAGACGTTGATACCCGCAAAAGGTT
>Common-cuckoo
CAGCACCGTGACCGAAGAGTCGGTTCGATCATCCTTGTAGTCTAATTCGCCAATCGCTACTAGCACTGTATGGCTAAATATACGCCAACTAAAAGGACGGCTGTGCTCGAACAGGTAAAGCAATATTAGGATCTTATGAGATAATTCTCTGCTCCTGAGGAGGTCACAAGTTGCTGGGATACTGATACTCACAAAAGGTT
>Carmine-bee-eater
CACCACGGCGACCGAAGATTCGGCTCGATCGTGCTTGTAATATAATTCGACAGGTGCTACCAGCACTGTATAGTTGAATACACATCAACTAAAAGGGCGGTTTCGCTCGAACAGGTACAGCAATATTAAGGTCTAACGAGATAGTTCTCTGCTTCTGAATAGGTCACACATTGCCGAGACATTGATACCCGCAAAAGGTT
>Downy-woodpecker
CAACACGGTGACGGAAGACTCGGTTCGATCTTCCTTGTACTATAATTCGACAAGCGCTACTAGTACTGCATGGTCGAATGCACATCAACTAGAAAGACGGTTCCGCTCGAACAGGTATAACGATCCTAAGGTTTTATTAGATAGTTCCCTGCTTCTGAGCAGATGATAAGTTGCTGAGACATTCATACCCGCAAAAGGTT
>Golden-collared-manakin
CACCTCGGTGGCCGAAGATTCAGTTTGATCGTCCTTGTACTATAATTCGACAAACCCTACTAGAACTGTATAGTTAAATACAAATGAACTAAAAAGACGGTCCCGCTCGAACAGGTACAGCAATATTAAGGTCTTATGAAATAGTTCTCGGCTTCTGAGTAGATCACAAGTCGCTGGGACATTGATACCGGAAAAAGGTT
>Great-crested-grebe
CACCGCGGTGGCTAAAGATTCGGTTCGATCGTCCTTATAGTATAATTCGACAGGCGCTACTAGCATTGTATAGTTAAACACGCATCAAGTAAAAGGACGGTTCTGCTCGAACAGGTACAGCAATATTAAGGTTTTATGAGATAGTTCACTGCTCCTGAGTAGATCACAAGTTACTGAGAAATTGATACCCGCAAAAGGGT
>White-tailed-eagle
CACCACGGTGGCCGAAGATTCGGTTCGATCGTCCTCGTACTATAATTCGACAAGCGCTACTAGCACTGTATAGTTAAATACACATCAACTAAAAGGACAGTTCCGCTCGAACAGCTACAGCAATATTAGGGTCTTATGAGATAGTTCTCTGCTACTGAGTAGATCACAAGTTGCTGAGACATTGATACCCGCAAAAGGTT
>Medium-ground-finch
CACCACGGTGGCTGAAGATTCGGTTTAATCGTCCTTGGACTATAATCCGACCAGCCATATTAGCACTGTATAGCTATATGCAAATAAACTAAAAGGACGATTATGCTCGGACAGGTACAGCAATATTAAGGTCTTACGAGATAGCTCTCGGCTTCTGAGTACATGACAAGGTGCTGGGACATTGATACCGGCAGAAAGTT
>Yellow-throated-sandgrouse
CACCACGATGGCCGAAGATTCGGTTCGATCGTCCCTGTACTATAATTCGACAAGCGATACTAGCACTGTATAGTTAAATACACATCAACCAAAAGGACGGTTCCGCCCGGAGAGGTACAGCAATATTAAGGTCTTATGAGATAGTTCTCTGCTCCTGAGTAAATCACGAGTTGCTGAGGCACTGATACCCGCAAAAGGTT
>Rhinoceros-hornbill
CACTACGGTGACCGAAGGTTTGGTTCTATCGTCCTTGTACTATAGTTCGACGAGCGCTACTAGCACTGTATAGTTAAATCCACATCAACTAAAAGGACGGTTCCGCTCGAACAGGTACAGTAATATTAAGGTCCTACGAGATAGTTTTCTGCTTCTGGGTGGATCACAAGTTGCCGAGACATGGATACCCGCAAAAGGTT
>Dalmatian-pelican
CACCACGGTGGCCAAAGATTCGGTTGGATAGTACTTGTACTATAATTCGACTACCCCTACTAGCATTGTATAGTTAAATACACATCAACTAAAAGGACCGTTCCGCTCGAAGAGGTACAGCGATATTAAGGTCTTATGAGATAGTTCTCTGCTTCTGAGTAGATCACAAGTTGCTGAGACAGTGATACCCGCAAAAGGTT
>Sunbittern
CACCACGATTGCCGCAGATTAGGTTAGATAGTCCTTGTACTATAATTCGACAAGCGTTACTAGCACTGTATGGTTAAATTCACATCAACTAAAAGGACGCTTCCGTTCGAACAAGCACAGTAATATTAAGGTCTTATGAGATGGTTCCTTGCTTCTTAGTAGACCACACGTTGTTGGGACATTGTTACCCGCAAAAGGTT
>Red-throated-loon
CACCACGGTGGCCGAAGATTCGGTTGGACCGTCTTTGTACTATAATTAGACAACCGCTACTAGCGCTGTATAGTTAAATACACATCAACTAAAAGCACGGTTCCGCTCGAACAGGGACAGCAATATTAATGTCTTATGAAATAGTTCTCTGCTTCTGAGTAGATCACAAGTTGCTGGGACAGTGATAACCGCAAAAGGTA
>American-crow
CACCACGGTGGTTGAAGATTCAATTTGATCGTCTTTGGAGTATAATTCGACCAGCCATACTAGCACTGTATAGTTATATGCAAATGGACTAAAAGGACGGTTCTGCTCGGACAGGTACAACAATATTAAGGTCTTATGAGATAGCTCTCGGCTTCTGAGTACATCATAAGTTGCTGGGACATTGATACCGGCAAAAGGTT
>Great-cormorant
CACCACGGTGGCCGACGATTCGGTTGAATCGACCTTGCACTATAAATCGACAACCAGTACCAGCGCTGTATAGTTAAATACACATCAACTTAAAGGACGGTTCTGCTCGAACAGGTACAGTAATATCAAGGACTTAAGAGATAGTTCTCTGCTTCTGATTAGATCACAAGTTGCTGAGACAGTGATACCCCCAAAAGGTT
>Chuck-wills-widow
CACGACGGTGGCCGAAGATTCGGTTCGATCGTCCTTGTACTATAACTCAACAAGCGCCACTAGCACTGTATAGTTAAATACACATTGACTAAAAGGACGGTTCCGCTCGGACAGATACAGCAGTATTAAAGCCTTATGAGATAGTTCTCTGCCCCTGAGCAGATCACAAGTTGTTGAGACATTAATACCCACAAAAGGTC
>Little-egret
CACCACGGTGGCCGGAGATTCGGTCGGATCATCCTTGTACTATAATTCGACAACCGCTACTAGCACTGCATAGTTAAATACACATCAACTAAAAGGACGGTTCCGCGCGAACAGGTACAGCAATATGAAGGCCCTATGAGATAGTTTTCTGCTTCTGAGTAGATCACAAGTTTCTGAGACAGTGATACTCGCAAAAGGTT
>White-throated-tinamou
CACCACGGCCGCTAACGATTCGGTTCGAGCGCCCTTGTACTATGATTCGTCAAGCCGGAGTAACACTGTTTAATTATATAGACCTCAAGTTACAGGACATCTCCACTTGAAGAGGTACAGCAATACTAAAATATTATGACGGAGTTCTCTGCTCCCGCGTGGACCATAAGTTACTCTGAGGTCGATACTTGCAGAAGGCT
>Zebra-finch
CACCACGGTGGCTGAAGATTCGGTTTAATCGTCCTTGGACTATAATTCGACCAGCGATACTAGCACTGTATAGCTATATGCAAATGAACTAAAAGGACGGTTATGCTCGGACAGGTACGGCAATATTAAGGTCTTATGGGATAGCTCTCGGCTTTTGAGTACATCACAAGGTGCTGGGACATTGATACTGGCAAAAGGCT
>Turkey-vulture
CACCACGGTGGCCGAAGATTCGGTTCGATCGTCCTTGTACTATAATTCGACAAGCGCTACTAGCACTGTATAGTTAAATACACATCAACTAAAAGAACGGTTCCGCTCGAACAGGTACAGCGATATTAGGGTCTTATGAGATAGTTCTCTGCTTCTGAGTAGATCACAAGTTGCTGAGACATTGATACCCGCAAAAGGTT
>Common-ostrich
CACCACGGCGGCCAACGATACGGTTCGATCGTCCTTGTACTATGATTCGACAAGCCGGACTAGCACTGTTTAGTTATATACACATGAACTAAAAGGACCTTTCCATTTGAAGAGGCACAGAAATATTAAAATCTTATGAGGTAGTTCTCTGCTCGCGGATAGACCATAGGTTACTGAAAGATTGATACTTGCAGAAGGCT
>Bar-tailed-trogon
TACCACGGTGACCGAAGATTCGGTTAGATCGTCCCTGTCCTATAATTCGACAAGTGCTACTTGCACTGTGTAGTTAAATACGGATCAACTAAAAGGACGGTTTAGCTCGAACTGATACAGAAATATTAAGGTCTTACGAGATTGTTATCTGCCTCTGAGTAGACCACAAGTTGCTAAGGCATTGATACCCGCACAAGGTT
>Rifleman
CACCACGGTGGCCAAAGATACGGTTTGATCGTCCTTGTACTATAATTCGACAAGCGCTACTAACACTGCATAATTAAATACCAATGAACTAAAAGGACGGTTCCGCTCTAACAGGTACAGCAATATTAAGGTCTTATGAAATAGTTCTCTGCTTCTGAGTAGATCACAAATTGCTGGGACATTGATACCGGCAAAAGGTT
>Chicken
CACCACAGTGGTCAGCGATCCGGTTCCATCGTCCTTGTACTATAATTCGACAAGCTTTACTAGCACTGTATAGTTAGATACACATCAACCAAAAGGACGTTTCCCCTCGAACAAGTACAGCAATAATAGTGTCATATTAGATAGTTTTTTGCTTGTGGGGAGATCACAAGTTACTGAGACGTAGATACCCGCAAAAGGTT
>American-flamingo
CACCACGGTGGCCGAAGATTCGGTTCGATCGTCCTTGTACTACAATTCGACAAGCGCTACTAGCACTGTATAGTTAAATACACATCAAATAAAAGGACGGTTCCGCTAGAACAGGTACAGCAATATTAAGGTCTTATGAGATATTTCTCTGCTCCTGAGTAGATCACAAGTTGCTGATACATTGATACCCGCGAAAGGGT
>Red-legged-seriema
CACCACGGTGGCCTAATATTCGGTTAGATCGTCCTTGTACTATAATTCGATAAGCGCTACTAGAACTGTATAGTTAAATACACATCAACTAAAAGGACGGTTCCACTCGAATAGGTACAGCAATATTAAGGTCTTATGAGACAGTTCTCGGCTTCTGATTAGATCACAAGTTGCTGAGACATTGATACCCGCAAAAGGTT
>Killdeer
CACCACGGTGGCCGAAGATTCAGTCCGATAGTCCTTGTACTCTATTTCGACAAGCGCTACTAGCACTGTATAGTTAAATACACATCAACTAAAAGGACGCTTCCGCTCGAACAGGTACAGCAATATTAAGGTCTTATGAGATAGTTGTCTGCTCCTGAGTAGATCATAAGTTACTGAGGCATTGATATCCGCAAAAGGTT
>Turkey
CACCACAGTGGTCAACGATCCGGTTCCATCGTCCTTGTACTATAATTCGACAAGCGTTACTAGCACTGTATAGTTAAATACACATCAACCAGAAAGACGTTTCCCCTCGAACAGGTACAGCAATAATAGTGTCTTATTAGATAGTTCTCTGCTTGTGGGGAGATCACAAGTTACTGAGACGTAGATACCCACAAAAGGTT
>Peregrine-falcon
CACCACGGTGGCCGAAGATTCGGTTCGCTCGTCCCTGTACTATAATTCGACAAGCGCTACTAGCACTGTATAGGTAAATACACATCAACTAAAAAGACGGTTCCGCTTGAAGAGGTACAGCCATATTAAGGTCTTATGAGATAGTTCTCGGCTTCTGAGTGGATCACAAGTTGCTGAGACATTGATACTCGCAAAAGGTT
>Emperor-penguin
CACCACGGTGGCCGCAGATTCGGTTGAATCGTCCTTGTACTATAACTCGACAACCGCTACTATCACTGTATAGTTAAATACACATCAACTAAAAGGACGGTTCAGCTCGAACAGGTACAGCAATATTAAGGTCTTATGCGATAGTTCTCTGCTTCTGAGTTGATCACAAGTTGCCGAGACAGTGATACCCTCAAAAGGTT
