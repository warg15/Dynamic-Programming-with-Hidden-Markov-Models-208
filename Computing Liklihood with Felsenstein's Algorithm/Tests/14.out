-3414.7886286073226
